### ISSUE TITLE
chance of ban for "platfrom abuse"?

### ISSUE THREAD

**Issue by Auncaughbove17** [basti564/Oculess#14]
What is the chance I will be banned for this?

**Comment by basti564** [comment]
Yes, pretty much zero. To this day no banned Quest (2) is known. FB only banned FB accounts that were obviously using fake names/info.
